cmake/projects/NASM/hunter.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017, Zhuhao Wang
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+if(CMAKE_HOST_WIN32)
+  # NASM only supports a subset of VS compilers on Windows. However, the only
```

Comment:
I haven't been using windows for quite a long time, but if I'm understanding this correctly, MinGW and Cygwin do not depend on VS. So it would be really too much hassle to require user who just want to build some package (e.g., OpenSSL) to install VS (note this is also a commercial software) just to build a program only needed in compiling phase. Of course it would be great if `add_hunter_package` can have some parameter to tune if we will build from source or just download pre-build binary.